### ISSUE TITLE
"Publishing .NET applications on Windows 10 is not yet supported"

### ISSUE THREAD

**Issue by ForNeVeR** [JetBrains/azure-tools-for-intellij#130]
Today a user contacted me and told that he was getting the error message "Publishing .NET applications on Windows 10 is not yet supported". That was surprising at least.

What happened actually was that he had selected a non-web project in the run configuration editor. We should improve the error message:

- show project type name if we cannot publish the project due to project type, e.g. "Publishing of Console applications is not supported", or maybe just leave generic "This project type cannot be published"
- show operating system only if we cannot publish due to wrong operating system (e.g. `!isWindows` currently): "Publishing for [this project type] is not yet supported on [OS]"

**Comment by sdubov** [comment]
Fixed in 183.1 branch. 

@ForNeVeR, currently, we have no ability to get project type from PublishableProjectModel. So, I've split cases to make sure we cover them separately. Create a new ticket to make it easily and more clear after we add a project kind into publishable project model -https://youtrack.jetbrains.com/issue/RIDER-23430.

Also, in scope of this ticker (initial problem was in the hidden "Project" combo box), I've added the logic to resize existing web apps table based on parent window height.
